test/e2e/apimachinery/webhook.go

```diff
@@ -71,12 +71,20 @@ var serverWebhookVersion = utilversion.MustParseSemantic("v1.8.0")
 
 var _ = SIGDescribe("AdmissionWebhook", func() {
 	var context *certContext
+	var ns string
+	var c clientset.Interface
 	f := framework.NewDefaultFramework("webhook")
 	framework.AddCleanupAction(func() {
-		cleanWebhookTest(f)
+		// Cleanup actions will be called even when the tests are skipped and leaves namespace unset.
```

Comment:
I thought the namespace is set up during f.BeforeEach, so len(ns) won't be 0 even if the test is skipped.

Reply:
```go
type Framework struct {
	Namespace                *v1.Namespace   // Every test has at least one namespace unless creation is skipped
}
```